src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { NavbarBrand } from './NavbarBrand';
+import { NavbarCollapse } from './NavbarCollapse';
+import { NavbarContext } from './NavbarContext';
+import { NavbarToggle } from './NavbarToggle';
+
+export type NavbarCompnentProps = {
+  menuOpen?: boolean;
+  fluid?: boolean;
+  rounded?: boolean;
+  border?: boolean;
+};
+
+const NavbarCompnent: FC<NavbarCompnentProps> = ({ children, menuOpen, fluid, rounded, border }) => {
+  const [isOpen, setIsOpen] = useState(menuOpen);
+
+  return (
+    <NavbarContext.Provider value={{ isOpen, setIsOpen }}>
+      <nav
+        className={classNames('border-gray-200 bg-white px-2 py-2.5 dark:bg-gray-800 sm:px-4', {
```

Comment:
Can you please add the dark color class of the border (ex: `dark:border-gray-700`)